### PR TITLE
Add tabletype tag to improve metrics granularity.

### PR DESCRIPTION
**Pull request by ArtfulCoder**

### CODE REVIEW

File: metacat-connector-hive/src/main/java/com/netflix/metacat/connector/hive/util/HiveTableUtil.java [hunk 1]
```diff
@@ -110,10 +112,20 @@ private static Deserializer createDeserializer(final Class<? extends Deserialize
...
+        final String tableTypeVal = getTableType(tableInfo);
+        return DirectSqlTable.ICEBERG_TABLE_TYPE.equalsIgnoreCase(tableTypeVal);
+    }
+
+    private static String getTableType(final TableInfo tableInfo) {
+        final Map<String, String> metadata = tableInfo.getMetadata();
+        if (metadata == null) {
+            return null;
+        }
+        final String tableType = metadata.get(DirectSqlTable.PARAM_TABLE_TYPE);
```
Comment (raveeram) [inline metacat-connector-hive/src/main/java/com/netflix/metacat/connector/hive/util/HiveTableUtil.java:124]: Do we also want to make sure the tableType here is non-null or empty?

File: metacat-main/src/main/java/com/netflix/metacat/main/api/RequestWrapper.java [hunk 2]
```diff
@@ -216,10 +219,23 @@ public QualifiedName qualifyName(final Supplier<QualifiedName> nameSupplier) {
...
         }
     }
 
+    private static void tryAddTableTypeTag(final Map<String, String> tags, final QualifiedName qualifiedName) {
+        final MetacatRequestContext context = MetacatContextManager.getContext();
+        if (context == null) {
+            return;
+        }
+        String tableType = context.getTableType(qualifiedName);
+        if (tableType == null) {
```
Comment (raveeram) [inline metacat-main/src/main/java/com/netflix/metacat/main/api/RequestWrapper.java:233]: probably make this null or empty. StringUtils.isBlank()

File: metacat-connector-hive/src/main/java/com/netflix/metacat/connector/hive/util/HiveTableUtil.java (hunk 1 above)
Comment (raveeram) [reply metacat-connector-hive/src/main/java/com/netflix/metacat/connector/hive/util/HiveTableUtil.java:124]: Nvm, saw your check below.

**Review by raveeram — APPROVED** [review]
Looks good abhishek!

File: metacat-main/src/main/java/com/netflix/metacat/main/api/RequestWrapper.java (hunk 2 above)
Comment (ArtfulCoder) [reply metacat-main/src/main/java/com/netflix/metacat/main/api/RequestWrapper.java:233]: fixed

**Review by raveeram — APPROVED** [review]

**Pull request merged**